diff --git a/node_modules/@notionhq/client/build/src/Client.js b/node_modules/@notionhq/client/build/src/Client.js
index 0b6d478..98c2f34 100644
--- a/node_modules/@notionhq/client/build/src/Client.js
+++ b/node_modules/@notionhq/client/build/src/Client.js
@@ -16,7 +16,7 @@ const logging_1 = require("./logging");
 const errors_1 = require("./errors");
 const utils_1 = require("./utils");
 const api_endpoints_1 = require("./api-endpoints");
-const node_fetch_1 = require("node-fetch");
+const node_fetch_1 = { get default() { return require("node-fetch").default; } };
 const package_json_1 = require("../package.json");
 class Client {
     constructor(options) {
